.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  extends: ['eslint-config-uber-es2015', 'prettier', 'plugin:react/recommended'],
+  parserOptions: {
+    ecmaVersion: 2018
+  },
+  rules: {
+    'guard-for-in': 0,
+    'no-inline-comments': 0,
+    camelcase: 0,
+    'sort-vars': 0
+  },
+  globals: {
+    performance: true,
```

Comment:
Regarding `global` and `env`. We don't want to disable these here, because we don't want code to access these globals, except in our specific wrapper that use these values. So better add local overrides in those files rather than remove warnings across the code base`

Reply:
eslint doesn't complain when I deleted this, so i guess they are not used